client/my-sites/email/inbox-management/index.js

```diff
@@ -0,0 +1,36 @@
+import { localize } from 'i18n-calypso';
+import React from 'react';
+import { connect } from 'react-redux';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import EmailManagementHome from 'calypso/my-sites/email/email-management/email-home';
+import { hasEmailSubscription } from 'calypso/my-sites/email/email-management/home/utils';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+const InboxManagement = ( { domains } ) => {
+	const nonWPCOMDomains = domains.filter( ( domain ) => ! domain.isWPCOMDomain );
+	const domainsWithSubscriptions = nonWPCOMDomains.filter( ( domain ) =>
+		hasEmailSubscription( domain )
+	);
```

Comment:
It looks like we could simplify this block of code like this:

```suggestion
	const domainsWithSubscriptions = domains.filter( ( domain ) => ! domain.isWPCOMDomain && hasEmailSubscription( domain ) );
```

Reply:
Right!